src/scss/main.scss

```diff
@@ -117,11 +117,11 @@ blockquote {
 }
 
 .alert--warning {
-    background: #ffc;
+    background: #ffffcc;
 }
 
 .alert--error {
-    background: #fcc;
+    background: #ffcccc;
```

Comment:
Curious to know why? Is the short code discouraged?

Reply:
My stylelint is still using the rules from Say Media, where long hex codes were preferred. The reason you see it here is because stylelint was broken and I fixed it while working on this branch, so I had to make some lint fixes.